weblate/vcs/git.py

```diff
@@ -656,18 +683,24 @@ def create_pull_request(self, origin_branch, fork_remote, fork_branch):
             head = fork_branch
         else:
             head = "{0}:{1}".format(fork_remote, fork_branch)
-        self.execute(
-            [
-                "pull-request",
-                "--force",
-                "--head",
-                head,
-                "--base",
-                origin_branch,
-                "--message",
-                self.get_merge_message(),
-            ]
+        pr_url = "{}/pulls".format(self.api_url())
+        r = requests.post(
+            pr_url,
+            headers={
+                "Accept": "application/vnd.github.v3+json",
+                "Authorization": "token {}".format(settings.GITHUB_TOKEN),
+            },
+            json={
+                "head": head,
+                "base": origin_branch,
+                "title": self.get_merge_message(),
+            },
         )
+        response = r.json()
+        if "url" not in response:
+            report_error(
+                cause="Pull request failed", extra_data={"errors": response["errors"]}
+            )
```

Comment:
This probably should gracefully handle pull request already exists case, see https://github.com/WeblateOrg/weblate/issues/4266

Reply:
So for Pull Request already exist case, I should not show any error, and report_error on other errors (and also probably raise error )?

Reply:
* Pull requests already exists should be silently ignored
* Anything else should probably raise an exception here. The `report_error` only logs it, but the failure should be made obvious here and raise an exception.